tests/models/registry.py

```diff
@@ -242,7 +247,8 @@ def check_available_online(
                                           {"tiny": "TitanML/tiny-mixtral"}),  # noqa: E501
     "QuantMixtralForCausalLM": _HfExamplesInfo("mistral-community/Mixtral-8x22B-v0.1-AWQ"),  # noqa: E501
     "MptForCausalLM": _HfExamplesInfo("mpt", is_available_online=False),
-    "MPTForCausalLM": _HfExamplesInfo("mosaicml/mpt-7b"),
+    "MPTForCausalLM": _HfExamplesInfo("mosaicml/mpt-7b",
+                                      trust_remote_code=True),
```

Comment:
This one is also not remote the custom config we used to have just did something weird. See https://github.com/vllm-project/vllm/pull/21931/files#diff-1768ac8a42d042ad6b9668ac812725aa0c716103957aa96266327eb8d3681d6a for the proper fix